### PR TITLE
chore: Adds router service (docker)

### PR DESCRIPTION
**Pull request by soluchok**
* adds router service (docker)
* fixes minor UI bug

Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: test/bdd/fixtures/agent-wasm/docker-compose.yml [hunk 1]
```diff
@@ -8,6 +8,23 @@ version: '2'
 
 services:
 
+  edge.router.agent.example.com:
+    container_name: aries-js-worker-edge-router.com
```
Comment (troyronda) [inline test/bdd/fixtures/agent-wasm/docker-compose.yml:12]: the “js-worker” bit can be dropped from the names.

File: test/bdd/fixtures/agent-wasm/.env [hunk 2]
```diff
@@ -11,6 +11,21 @@
 
 # Agent configurations
 USER_WASM_IMAGE=docker.pkg.github.com/trustbloc/edge-agent/user-agent-wasm
+AGENT_REST_IMAGE=aries-framework-go/agent-rest
+AGENT_REST_IMAGE_TAG=latest
```
Comment (rolsonquadras) [inline test/bdd/fixtures/agent-wasm/.env:15]: Better to have a specific tag than `latest`.

**Comment by rolsonquadras** [comment]
@soluchok the commit is not signed.

File: test/bdd/fixtures/agent-wasm/.env (hunk 2 above)
Comment (soluchok) [reply test/bdd/fixtures/agent-wasm/.env]: Added `amd64-0.1.3`. I am not able to use `0.1.4-snapshot-fa7a392`. Any idea why?

**Pull request merged**